out/index.html

```diff
@@ -0,0 +1,1273 @@
+<!doctype html>
```

Comment:
Please don't check generated code into the source repository.

Reply:
Looks like this slipped through after testing. I'll fix it.